client/packages/inventory/src/Stocktake/ListView/ListView.tsx

```diff
@@ -44,6 +44,8 @@ export const StocktakeListView = () => {
   } = useUrlQueryParams();
   const pagination = { page, first, offset };
   const { data, isError, isLoading } = useStocktakeOld.document.list();
+  const { data: hasStocktake, isLoading: firstStocktakeLoading } =
```

Comment:
Why couldn't you reuse the list() api call? 🤔

Reply:
It's affected by the URL parameters. I could probably have used it and somehow over-ride the URL params in a second hook? Never done that before but definitely would be nice to keep it simple?

Reply:
Ah right yeah I see now~